src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/UsersController.cs

```diff
@@ -80,5 +81,28 @@ public async Task<ActionResult> CreateUserAsync(Guid organisationId, CreateUserR
 
             return Ok();
         }
+
+        [Route("users/{userId}")]
+        [HttpGet]
+        public async Task<ActionResult> GetUserById(string userId)
```

Comment:
The return result can make use of the generic version of `ActionResult`, i.e. `Task<ActionResult<GetUserViewModel>>`. One of the benefits of that is  it reduces the number of Swagger annotations required (if we get round to enabling it).